modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -204,6 +204,54 @@ def get_facility_timezone_memoized(facility_location_id)
         facility_info[:timezone]&.[](:time_zone_id)
       end
 
+      def normalize_eps_appointment(appt)
+        {
+          id: appt[:id].to_s,
+          status: map_state_to_status(appt[:state]),
+          patientIcn: appt[:patientId],
+          created: appt.dig(:appointmentDetails, :lastRetrieved),
+          requestedPeriods: [
+            {
+              start: appt.dig(:appointmentDetails, :start),
+              end: calculate_end_time(appt.dig(:appointmentDetails, :start))
+            }
+          ].compact,
+          locationId: appt[:locationId],
+          clinic: appt[:clinic],
+          contact: appt[:contact]
+        }.compact
+      end
+
+      # rubocop:disable Lint/DuplicateBranch
+      def map_state_to_status(state)
+        case state&.downcase
+        when 'draft'
+          'proposed'
+        when 'submitted'
+          'booked'
+        else
+          'proposed'
+        end
+      end
+
+      # rubocop:enable Lint/DuplicateBranch
+
+      def calculate_end_time(start_time)
+        return nil unless start_time
+
+        Time.zone.parse(start_time) + 60.minutes
+      end
+
+      def merge_appointments(new_appointments, basic_appointments)
```

Comment:
I feel that 'new_appointments' and 'basic_appointments' might lead to confusion as to what makes an appt a new vs a basic. Would it make sense to call them what they are, which is eps_appointments and vaos_appointments?

Reply:
To keep renaming to a minimum, I'm going to change it to `eps_appointments` and change `basic_appointments` back to appointments